pipeline/enigma/enigma-processing.py

```diff
@@ -1,93 +1,145 @@
 """
```

Comment:
This file needs review to determine which version is more correct. I believe Lucy is also working on this file.